src/api/index.ts

```diff
@@ -1,4 +1,5 @@
-import Database, { Database as DatabaseInstance } from 'better-sqlite3'
+// @ts-ignore
+import type { Database } from 'better-sqlite3'
```

Comment:
we only want to work with the type, see https://github.com/digidem/mapeo-map-server/pull/93#discussion_r1092495401